cmake/FindLLVM.cmake

```diff
@@ -118,3 +118,4 @@ endif()
 enable_language(C CXX)
 # required by ROCm-Device-Libs, must be after project() call
 find_package(LLVM REQUIRED CONFIG NO_DEFAULT_PATH PATHS ${CLANG_ROOT_PATH}/lib/cmake/llvm)
+find_package(Clang REQUIRED CONFIG NO_DEFAULT_PATH PATHS ${CLANG_ROOT_PATH}/lib/cmake/clang)
```

Comment:
Why NO_DEFAULT_PATH is there? It doesn't find it here if I have it set.

Reply:
We want to make sure we use the same Clang that's associated with the `llvm-config` that the user provided. many systems have a pre-installed Clang and we want to make sure we use matching versions.

This again would indicate to me that your Clang was built in some different way if LLVM cmake config and Clang cmake config reside in different directories.

Reply:
OK. I'll investigate more.